driving-game/main.js

```diff
@@ -0,0 +1,15 @@
+var $car = document.querySelector('.car');
+
+window.addEventListener('keydown', rotateCar);
+
+function rotateCar(event) {
+  if (event.key === 'ArrowDown') {
+    $car.className = ('class', 'car down');
```

Comment:
I'm not sure how this is working, but you should just assign a string with both classes separated by a space.